spec/forms/state_file/nj_eligibility_health_insurance_form_spec.rb

```diff
@@ -16,18 +20,69 @@
 
       expect(form.errors[:eligibility_all_members_health_insurance]).to include "Can't be blank."
     end
+
+    # TODO: add validations that *someone* is checked if we add a box for other/self/spouse
```

Comment:
[DUST] (weakly held) Same as above, prefer to remove TODO comments and capture them in a ticket instead. Reasoning would be that we might not actually end up adding a box for other/self/spouse